src/main/drivers/accgyro_mpu.c

```diff
@@ -129,11 +129,11 @@ static void mpuIntExtiHandler(extiCallbackRec_t *cb)
 static void mpuIntExtiInit(gyroDev_t *gyro)
 {
 #if defined(MPU_INT_EXTI)
-    if (!gyro->mpuIntExtiConfig) {
+    if (gyro->mpuIntExtiConfig.tag == IO_TAG_NONE) {
```

Comment:
Personally I prefer use tag as boolean, reading it as _if there is no gyro->mpuIntExtiConfig_ .. but both ways work